controller/search.go

```diff
@@ -83,6 +83,12 @@ func (c *SearchController) Show(ctx *app.ShowSearchContext) error {
 
 	}
 	return application.Transactional(c.db, func(appl application.Application) error {
+		if ctx.Q == nil || *ctx.Q == "" {
+			jerrors, _ := jsonapi.ErrorToJSONAPIErrors(
```

Comment:
Yea, it's not the best to log all request bodies... but we can evaluate as we go if we want to log additional things on certain errors. Would have love to have a complete 'stream history with reply abillity tho' but you know.. :)